app/router/routes.py

```diff
@@ -0,0 +1,26 @@
+import sys
+import os
+
+sys.path.insert(
+    0, os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+)  # Fixes import errors
+
+from fastapi import APIRouter, HTTPException
```

Comment:
Related to the suggestion below
```suggestion
from fastapi import APIRouter, HTTPException, status
```